src/style.rs

```diff
@@ -110,6 +110,60 @@ impl fmt::Debug for Modifier {
     }
 }
 
+#[cfg(feature = "crossterm")]
+impl From<crossterm::style::Attribute> for Modifier {
+    fn from(value: crossterm::style::Attribute) -> Self {
+        use crossterm::style::Attributes as CAttributes;
+
+        Modifier::from(CAttributes::from(value))
+    }
+}
+
+#[cfg(feature = "crossterm")]
+impl From<crossterm::style::Attributes> for Modifier {
+    fn from(value: crossterm::style::Attributes) -> Self {
+        use crossterm::style::Attribute as CAttribute;
+        let mut res = Modifier::empty();
+
+        // Note CAttribute::Reset is not here because it corresponds to an empty Modifier.
+
+        if value.has(CAttribute::Bold) {
+            res |= Modifier::BOLD;
+        }
+        if value.has(CAttribute::Dim) {
+            res |= Modifier::DIM;
+        }
+        if value.has(CAttribute::Italic) {
+            res |= Modifier::ITALIC;
+        }
+        if value.has(CAttribute::Underlined)
+            || value.has(CAttribute::DoubleUnderlined)
+            || value.has(CAttribute::Undercurled)
+            || value.has(CAttribute::Underdotted)
+            || value.has(CAttribute::Underdashed)
```

Comment:
Aside: If these really work, do you think we should add them to Ratatui at some point?

Reply:
I think we either implement everything (see [crossterm](https://docs.rs/crossterm/latest/crossterm/style/enum.Attribute.html) and [ANSI wiki](https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters)) or stay as is.

It seems those are rarely supported features. For example my `xfce4-terminal` supports underlined and double underlined bot not fraktur or framed. And tbh `Underdotted`, `Undercurled` and `Underdashed` doesn't even show up on the wiki.

tldr; overall no one has requested that, let's keep it as is and implement it later if really needed